### PR TITLE
CLI for Hydrus & fixes for dependencies installation

### PR DESCRIPTION
**Pull request by IamRaviTejaG**
Fixes issues #64 & #102.

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description
The CLI can be used to start the server and currently supports the following options:
`-u` / `--adduser`: Adds user to the server.
`-d` / `--hydradoc`: Sets the HydraDoc file location.
`-a` / `--api`: Sets the API name.
`-p` / `--port`: Sets the API port.
`--auth / --no-auth`: Toggles the authentication.

### Test Logs
test-logs

### CODE REVIEW

**Comment by suryatejreddy** [comment]
@IamRaviTejaGI have created a PR on your fork resolving conflicts. Verify once and merge if there are no errors. Thanks

File: cli.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import click
+
+@click.command()
+@click.option("--adduser", "-u", default=tuple([1, "test"]),
+                help="Adds a new user to the API.", nargs=2, type=(int, str))
+@click.option("--api", "-a", default="serverapi",
+                help="The API name.", type=str)
+@click.option("--auth/--no-auth", default=True,
+                help="Set authentication to True or False.")
+@click.option("--hydradoc", "-d", default="doc.json",
```
Comment (xadahiya) [inline cli.py]: Should be doc.jsonld

**Comment by IamRaviTejaG** [comment]
@xadahiya Done renaming! Please review. :)

**Comment by xadahiya** [comment]
@IamRaviTejaG  change it in cli.py as well.

**Comment by IamRaviTejaG** [comment]
@xadahiya Done.

**Comment by IamRaviTejaG** [comment]
@Mec-iS Please review the changes. :)

**Comment by Mec-iS** [comment]
@xadahiya and @chrizandr have the last word for merging

**Comment by xadahiya** [comment]
Already approved the pr

**Pull request merged**